### ISSUE TITLE
ModuleNotFoundError: No module named 'pkg_resources'

### ISSUE THREAD

**Issue by phayes** [certbot/certbot#9765]
## My operating system is (include version): 

MacOS 13.4.1

## I installed Certbot with (snap, OS package manager, pip, certbot-auto, etc):

`brew install certbot`

## I ran this command and it produced this output:

`sudo certbot certonly --standalone`

```

Traceback (most recent call last):
  File "/opt/homebrew/bin/certbot", line 5, in <module>
    from certbot.main import main
  File "/opt/homebrew/Cellar/certbot/2.6.0_1/libexec/lib/python3.11/site-packages/certbot/main.py", line 6, in <module>
    from certbot._internal import main as internal_main
  File "/opt/homebrew/Cellar/certbot/2.6.0_1/libexec/lib/python3.11/site-packages/certbot/_internal/main.py", line 28, in <module>
    from certbot import configuration
  File "/opt/homebrew/Cellar/certbot/2.6.0_1/libexec/lib/python3.11/site-packages/certbot/configuration.py", line 11, in <module>
    from certbot import util
  File "/opt/homebrew/Cellar/certbot/2.6.0_1/libexec/lib/python3.11/site-packages/certbot/util.py", line 25, in <module>
    from certbot._internal import constants
  File "/opt/homebrew/Cellar/certbot/2.6.0_1/libexec/lib/python3.11/site-packages/certbot/_internal/constants.py", line 6, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
```

**Comment by phayes** [comment]
This was solved by running "python3 -m ensurepip"

Perhaps this command should be part of the homebrew recipe

**Comment by frenzymadness** [comment]
This is probably not necessary anymore as certbot should not use `pkg_resources`, see: https://github.com/certbot/certbot/commit/cc359dab469281fec10357c81b9dd2d4e06290b3 

But the change is not yet released.

**Comment by osirisinferi** [comment]
As of #9797 Certbot has removed all things related to `pkg_resources`.

It will land in the next release (probably 2.8.0).

Note that the earlier mentioned commit landed in 2.7.0 already.
